docs/etcdStructure.md

```diff
@@ -9,148 +9,233 @@
-	      |  `--peer_v6   # Global IPv6 BGP peers (all nodes peer with)
-	      |     `--<BGP peer IPv6 address>  # JSON BGP peer configuration (see below)
-	      `--host
-	         `--<hostname>  # one for each Docker host in the cluster
-	            |--ip_addr_v4 # the IP address BIRD listens on
-	            |--ip_addr_v6 # the IP address BIRD6 listens on
-	            |--as_num     # the AS number for this host
-	            |--peer_v4    # Host specific IPv4 BGP peers
-	            |  `--<BGP peer IPv4 address>  # JSON BGP peer configuration (see below)
-	            `--peer_v6  # Host specific IPv6 BGP peers
-	               `--<BGP peer IPv6 address>  # JSON BGP peer configuration (see below)
+    
+    +--calico  # root namespace
+       |
+       |--v1
+       |  |--config
+       |  |  |--InterfacePrefix # the prefix for Calico interface names
+       |  |  `--LogSeverityFile # Log severity level for writing to file e.g. "DEBUG"
+       |  |--host
+       |  |  `--<hostname>      # one for each Docker host in the cluster
+       |  |     |--config       # Host level config
+       |  |     |  `--marker
+       |  |     |--ipv4_address # the next hop IP address of this host
+       |  |     `--workload
+       |  |        `--docker
+       |  |           `--<container-id>  # one for each container on the Docker Host
+       |  |              `--endpoint
+       |  |                 `--<endpoint-id>  # JSON endpoint config (see below)
+       |  |--policy
+       |  |  `--profile
+       |  |     `--<profile-id>  # Unique string name
+       |  |        |--tags  # JSON list of tags
+       |  |        `--rules  # JSON rules config (see below)
+       |  `--ipam  #IP Address Management
+       |     |--v4
+       |     |   `--pool
+       |     |      `--<CIDR>  # One per pool, key is CIDR with '/' replaced
+       |     |                 # by '-', value is JSON IP Pool object (see below)
+       |     `--v6
+       |         `--pool
+       |            `--<CIDR>  # One per pool, key is CIDR with '/' replaced
+       |                       # by '-', value is JSON IP Pool object (see below)
+       |--ipam/v2 # IPAM configuration and assignment data
+       |  |--assignment
+       |  |  |--ipv4
+       |  |  |  `--block
+       |  |  |      `--<CIDR>  # One per block, key is CIDR with '/' replaced
+       |  |  |                 # by '-', value is JSON Allocation Block object (see below)
+       |  |  `--ipv6  
+       |  |     `--block
+       |  |         `--<CIDR>  # One per block, key is CIDR with '/' replaced
+       |  |                    # by '-', value is JSON Allocation Block object (see below)
+       |  |--handle
+       |  |  `--<Handle ID>    # One per handle, value is JSON Allocation Handle
+       |  |                    # object (see below)
+       |  |--host
+       |  |  `--<hostname>
+       |  |     |--ipv4
+       |  |     |  `--<CIDR>   # CIDR matching the Allocation Block with this host
```

Comment:
This and `ipv6` CIDRs are both one layer deeper under `block:
`/calico/ipam/v2/host/7fd0dabeeb24/ipv4/block/192.168.1.0-26`